judging/generate_predictions_local.py

```diff
@@ -50,6 +55,20 @@ def generate_predictions(requested_predictions_df: DataFrame, prediction_module:
             ]
         )
 
+        LOGGER.info(
+            f'Running validation module {validation_module} for {start_date} to {end_date} '
+            f'ip file {ip_file} output {output_file}')
+
+        # Now run validation
+        subprocess.call(
```

Comment:
Extra call to invoke validator. Simpler to do it here as we have all the params ready to go